src/components/HomePageButton.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+function HomePageButton() {
+  return (
+    <div>
```

Comment:
Probably better to wrap in a Fragment rather than a `<div>` if there aren't any styles being added to it.